include/linker-defs.h

```diff
@@ -32,6 +32,14 @@
 #include <toolchain.h>
 #include <sections.h>
 
+#ifdef CONFIG_FIRMWARE_VERSION_STRING
+#define VERSION_SECTION_SIZE 32
```

Comment:
Use this to define the start address. E.g. add a new define for the last address in Flash (or maybe it's already there). then the start address of the VERSION section is "FLASH_END - VERSION_SECTION_SIZE"